sway/tree/workspace.c

```diff
@@ -334,16 +334,13 @@ char *workspace_next_name(const char *output_name) {
 	if (target != NULL) {
 		return target;
 	}
-	// As a fall back, get the current number of active workspaces
-	// and return that + 1 for the next workspace's name
-	int ws_num = root->outputs->length;
-	int l = snprintf(NULL, 0, "%d", ws_num);
-	char *name = malloc(l + 1);
-	if (!sway_assert(name, "Could not allocate workspace name")) {
-		return NULL;
-	}
-	sprintf(name, "%d", ws_num++);
-	return name;
+	// As a fall back, use the next available number
+	char name[12] = "";
+	unsigned int ws_num = 1;
+	do {
+		sprintf(name, "%u", ws_num++);
```

Comment:
Can we use `snprintf` here, just in case?

Reply:
Done